src/components/ProfileBioModule.js

```diff
@@ -0,0 +1,67 @@
+import React, { Component, PropTypes } from 'react'
+import { updateUserSettings } from '../actions'
+import { trackEvent, ADDED_BIO } from '../util/analytics'
+
+export default class ProfileBioModule extends Component {
+  static propTypes = {
+    person: PropTypes.object
+  }
+
+  static contextTypes = {
+    dispatch: PropTypes.func
+  }
+
+  constructor (props) {
+    super(props)
+    const firstName = props.person.name.split(' ')[0]
+    this.state = {
+      firstName,
+      length: 0,
+      minLength: 1,
```

Comment:
`minLength` and `maxLength` can just be constants defined at the top level of this file, since they don't change